src/components/menu/menu.js

```diff
@@ -24,11 +23,9 @@ export default function Menu({
   return (
     <List $ref={rootRef}>
       {items.map((item, index) => {
+        const requiredProps = getRequiredItemProps(item, index);
         // $FlowFixMe
-        const {key, ...itemProps} = mapStyletronProps(
-          getRequiredItemProps(item, index),
-          STYLETRON_PROP_MAPPER,
-        );
+        const {key, ...itemProps} = mapStyletronProps(requiredProps);
```

Comment:
do we still want styletron to be optional in base ui? if so, we should mention it only in `styled-components` file

Reply:
good call! i'm not sure if this mapper is needed anymore tbh, how are we handling passing of styletron props down to styled components elsewhere - are we just hand mapping say `isHighlighted` to `$isHighlighted`?

Reply:
good point oleg, i'd agree styletron should ideally only be mentioned in styled.js. That said, you could still have this function and rename it to something unbranded like `mapStyleProps`.

I've just been doing the mapping manually:
```js
getSharedProps(): $Diff<SharedStylePropsArgT, {children: React.Node}> {
    const {isOpen, showArrow} = this.props;
    const {isAnimating, arrowOffset, popoverOffset, placement} = this.state;
    return {
      $showArrow: Boolean(showArrow),
      $arrowOffset: arrowOffset,
      $popoverOffset: popoverOffset,
      $placement: placement,
      $isAnimating: isAnimating,
      $isOpen: isOpen,
    };
}
```
But no reason _not_ to use a helper function if it makes sense for a given component.

Reply:
It seems to me that these functions all basically do a mapping of `{someKey: value}` => `{$someKey: value}`. Should we make this a helper?